app/services/dashboard/data.rb

```diff
@@ -57,7 +57,7 @@ def fetch_data
     if out_of_sync_enabled?
       @report.update({
                        :out_of_sync_hosts         => out_of_sync_hosts.count,
-                       :out_of_sync_hosts_enabled => out_of_sync_hosts.alerts_enabled.count,
+                       :out_of_sync_hosts_enabled => out_of_sync_hosts.alerts_enabled.count
```

Comment:
Seems to be an unrelated fix that should be in a different commit.

Reply:
this got in in the meanwhile, removed here.